src/requestor/arborist.py

```diff
@@ -167,3 +178,13 @@ async def grant_user_access_to_policy(
         logger.error(f"Unable to grant access, got status code: {status_code}")
 
     return status_code == 204
+
+
+async def revoke_user_access_to_policy(
+    arborist_client: ArboristClient,
+    username: str,
+    policy_id: str,
+) -> bool:
+    logger.debug(f"Attempting to revoke {username}'s access to {policy_id}")
+    success = await arborist_client.revoke_user_policy(username, policy_id)
+    return success == True
```

Comment:
Would it be better if we use `bool(success)` instead of `success==True`? I mean ultimately they both mean the same thing. the former looks more formal IMO.

Reply:
i like the explicit comparison better, there's no room for uncertainty about the type of the returned value.
this function returns either `None` or `True` so there's already uncertainty https://github.com/uc-cdis/gen3authz/blob/9f7f5fa7262e4d31f67f7f6497b5f589458bacaf/python/src/gen3authz/client/arborist/base.py#L703

Reply:
i would rather check for the exact value i'm looking for but it's just personal preference